cfpb/cfgov-refresh
# Megamenu changes for Mortgage Performance Trends launch

With the addition of Mortgage Performance Trends, the Data & Research megamenu entries have been re-ordered. This change has been approved by the content owners for this section of the megamenu.

This PR makes the new display contingent on the Mortgage Performance Trends launch feature flag being turned on. I consulted on the method for making this change with @higs4281 and @virginiacc.

### Current status (October 12, 2017)

![screen shot 2017-10-12 at 3 09 00 pm](https://user-images.githubusercontent.com/1183435/31515528-73776d56-af63-11e7-9c1f-e1b90853947f.png)

### After Mortgage Performance Trends launch

![screen shot 2017-10-12 at 3 29 19 pm](https://user-images.githubusercontent.com/1183435/31515534-78120ba0-af63-11e7-9220-980cd19978ba.png)

## Additions

- Wrap the future Data & Research menu items in a positive condition for the feature flag

## Changes

- Wrap the current Data & Research menu items in a negative condition for the feature flag

## Testing

1. Pull down this branch and `./setup.sh`
2. Set up the [mortgage performance API](https://github.com/cfpb/cfgov-refresh/pull/3189).
3. Create a [mortgage chart block, mortgage map block, and page](https://github.com/cfpb/cfgov-refresh/pull/3260).
4. Open the megamenu on the mortgage page. You should see the current megamenu entries that are on consumerfinance.gov as of October 12, 2017.
5. Go into the flags section of your local Wagtail admin, and set the `MORTGAGE_PERFORMANCE_RELEASE` flag condition to Boolean equals `True`. Save the condition, return to the mortgage trends page, and refresh it. You should now see the megamenu entries for after the Mortgage Performance Trends launch.

## Checklist
